jupyterlab/federated_labextensions.py

```diff
@@ -102,7 +102,7 @@ def develop_labextension(  # noqa
     full_dest = normpath(pjoin(labext, destination))
     if overwrite and os.path.lexists(full_dest):
         if logger:
-            logger.info("Removing: %s" % full_dest)
+            logger.info(f"Removing: {full_dest}")
```

Comment:
For reference, there was some discussion about using f-strings with the Python logger in the Voila repo, with could lead to potential performance issues: https://github.com/voila-dashboards/voila/pull/419#issuecomment-561089304

Some more info here: https://realpython.com/python-f-strings/#lazy-evaluation-in-logging

Maybe there is a way to configure ruff to not format the strings used with the logger, if we want to follow the recommendation of not interpolating the f-strings when it's not necessary?

Reply:
@jtpio You mean `logger.info("Removing: %s", full_dest)` vs `logger.info(f"Removing: {full_dest}")`, isn't?

If this is the case the two forms are not flagged by Ruff. Here it is flagged because `"Removing %s" % full_dest` actually does not use lazy evaluation.

Reply:
Oh right, just noticed we were using `"Removing: %s" % full_dest` and not `"Removing: %s", full_dest` inside the parentheses. Which is indeed probably why Ruff formatted it.

I guess we can then leave that for another PR (deciding which approach to use for the logging module).

Reply:
@jtpio just checking we can move forward and leave the log concern as follow-up?

Reply:
Ah yes, thanks for the reminder :+1: